packages/coin-base/src/types.ts

```
@@ -1,27 +1,26 @@
-import BigNumber from "bignumber.js";
 import { Transform, Type } from 'class-transformer';
 import { WalletSignatureSchema, Ecosystem } from "./enums";
 
 export class Chain {
-  value: BigNumber;
+  value: BigInt;
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)